core/src/main/scala/org/apache/pekko/persistence/jdbc/query/javadsl/CurrentLastKnownSequenceNumberByPersistenceIdQuery.scala

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Can you change the license header on new files that are not copies of other files to be https://github.com/apache/pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16 ?

Reply:
Sure! Updated in <https://github.com/apache/pekko-persistence-jdbc/pull/267/commits/b033218f3f11f813fffa6cb8c5f7a46b82022d2f>.